packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -407,6 +409,30 @@ class SemanticsProperties extends DiagnosticableTree {
   /// Doing so instructs screen readers to not read out the [value].
   final bool obscured;
 
+  /// If non-null, whether the node corresponds to a visually distinct subtree.
+  ///
+  /// When the most specific route semantics node changes, the platform
+  /// accessibility framework will make an edge-triggered announcement and clear
+  /// the current accessibility focus. Specificity is defined as the last child
+  /// in inverse hit-test order.
+  ///
+  /// See also:
+  ///
+  ///  * [routeName] for a description of how the announced value is selected.
+  final bool route;
+
+  /// If non-null, whether the node contains the semantic name for a route.
+  ///
+  /// When the most specific route semantics node changes, the platform will
```

Comment:
"When the most specific route semantics node changes" could be interpreted many different ways. We should phrase this in a less ambiguous way.